wagtail/wagtailadmin/static_src/wagtailadmin/scss/layouts/404.scss

```diff
@@ -0,0 +1,101 @@
+@import 'wagtailadmin/scss/variables';
+@import 'wagtailadmin/scss/mixins';
+@import 'wagtailadmin/scss/grid';
+@import 'wagtailadmin/scss/components/logo';
+
+body {
+    font-family: $font-sans;
+    color: $color-white;
+
+    .page404__bg {
+        position: fixed;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        background-color: $color-teal-darker;
+
+        .wagtail-logo-container__desktop {
+            position: fixed;
+            right: calc(50% + 130px);
+            top: calc(45% - 250px);
+            width: 400px;
+            height: 500px;
+
+            .wagtail-logo {
+                width: 100%;
+                height: 100%;
+            }
+        }
+
+        .page404__text-container {
+            position: fixed;
+            left: calc(50% - 130px);
+            top: calc(45% - 150px);
+            width: 600px;
+            text-align: center;
+            z-index: 50;
+
+            .page404__text--header {
+                position: relative;
+                top: -80px;
+                color: $color-white;
+                font-size: 80px;
+                padding-bottom: 0;
+            }
+
+            .page404__text--info {
+                position: relative;
+                top: -130px;
+                font-size: 30px;
+                line-height: 125%;
+            }
+
+            .page404__link--goback {
+                position: relative;
+                top: -125px;
+                font-size: 25px;
+                background-color: $color-teal-dark;
+                text-decoration: none;
+                color: #fff;
+                padding-left: 12px;
+                padding-right: 12px;
+                padding-top: 3px;
+                padding-bottom: 3px;
+                border-radius: 20px;
+            }
+        }
+    }
+}
+
+// SMALL DESKTOP CHANGES:
+@media screen and (max-width: $breakpoint-desktop-small) {
+    body {
+        .page404__bg {
+            .wagtail-logo-container__desktop {
+                display: none;
+            }
+
+            .page404__text-container {
+                left: calc(50% - 300px);
+            }
+        }
+    }
+}
+
+// MOBILE CHANGES:
+@media screen and (max-width: $breakpoint-mobile) {
+    body {
+        .page404__bg {
+            .page404__text-container {
+                width: 400px;
+                left: calc(50% - 200px);
+
+                .page404__text--header {
+                    font-size: 60px;
+                    padding-bottom: 30px;
```

Comment:
I don't think any of those styles would be necessary if the positioning wasn't done with exact numbers? Also, no nesting please – here this is going to generate a `body .page404__bg .page404__text-container .page404__text--header {` selector, which is completely unnecessary.